### PR TITLE
Feature frontend skeleton

### PR DESCRIPTION
**Pull request by ChianHuei**
a draft of landing page and shoppingList page
Screen Shot 2020-07-02 at 2 36 55 PM
Screen Shot 2020-07-02 at 2 36 40 PM

### CODE REVIEW

File: client/src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        <Link color="inherit" href="https://material-ui.com/">
+          BigDeal
+        </Link>{' '}
+        {new Date().getFullYear()}
+        {'.'}
+      </Typography>
+    );
+  }
+
+const footers = [
```
Comment (sina-jamshidi) [inline client/src/components/Footer.js]: Lol we can probably remove this extra stuff

**Review by sina-jamshidi — APPROVED** [review]
Looks good!

File: client/src/components/Footer.js (hunk 1 above)
Comment (ChianHuei) [reply client/src/components/Footer.js]: sure! I will remove it.

**Comment by Ethorf** [comment]
Looks absolutely fantastic so far!

**Review by surhud004 — COMMENTED** [review]
Looks great as far as my React knowledge lol

**Pull request merged**